buildpacks/jvm-function-invoker/Cargo.toml

```
@@ -0,0 +1,25 @@
+[package]
+name = "jvm-function-invoker-buildpack"
+version = "0.1.0"
```

@hone What should the version here be? It's not going to stay in sync with the actual buildpack version, and since we're not publishing as a crate it mostly doesn't matter?

---

```suggestion
version = "0.0.0"
```

Let's make this a more clearly non-valid version number, so it's not confused with the other buildpack version. In the future if the automation can update this version at the same time, then we can make it a real version :-)